mongo/version_store.go

```diff
@@ -233,14 +233,26 @@ func (m *Mongo) GetAllStaticVersions(ctx context.Context, datasetID, state strin
 	if state != "" {
 		selector["state"] = state
 	}
-	// get total count and paginated values according to provided offset and limit
+
 	results := []*models.Version{}
-	totalCount, err := m.Connection.Collection(m.ActualCollectionName(config.VersionsCollection)).Find(ctx, selector, &results,
+
+	// Build options dynamically to avoid "limit=0" meaning "no results"
```

Comment:
I'm interested to know why this has been changed and no tests added given it now completely changes the behaviour of the GET /editions endpoint for static datasets, and also means it has a completely different behaviour to all of our standard endpoints.